include/s11/s11_options.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019-present Open Networking Foundation
```

Comment:
is this file required ?

Reply:
s11 was currently using options file from s1ap. this file removes the dependency between s1ap and s11 and can be used for common functions needed in s11.